agogosml_cli/cli/templates/{{cookiecutter.PROJECT_NAME_SLUG}}/input_reader/main.py

```diff
@@ -22,11 +22,11 @@
                     'EVENT_HUB_SAS_POLICY': os.getenv("EVENT_HUB_SAS_POLICY"),
                     'EVENT_HUB_SAS_KEY': os.getenv("EVENT_HUB_SAS_KEY"),
                     'LEASE_CONTAINER_NAME': os.getenv("LEASE_CONTAINER_NAME"),
-                    'APP_HOST': os.getenv('APP_HOST'),
-                    'APP_PORT': os.getenv('APP_PORT'),
                     'TIMEOUT': os.getenv('TIMEOUT')
                 }
-            }
+            },
+            'APP_HOST': os.getenv('APP_HOST'),
```

Comment:
why is this change needed? why for the kafka type the same change is not needed?
if it does, consider moving this change before the 'if' statement.

Reply:
In the [input_reader_factory](https://github.com/Microsoft/agogosml/blob/master/agogosml/agogosml/reader/input_reader_factory.py), the app_host and app_port variables are accessed from the top level of the config, not within config.client. 

I still need to test the entire Kafka flow ... there are a few changes that need to be made for that. But want to at least fix this first.

Reply:
@eladiw also the CLI failures are due to a dependency issue with pyinstaller